src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,36 @@
 import React from 'react';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import { PostInfoType } from '../../type/post';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+export const PostInfo: React.FC<PostInfoType> = ({
+  post: {
+    id,
+    title,
+    body,
+    user,
+    comments,
+  },
+}) => (
+  <div className="PostInfo" key={id}>
```

Comment:
Remove the key attribute from the div element, as it should be used in the parent component where the PostInfo components are rendered in a loop.